packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -27,7 +27,7 @@ export function PayloadReplacer({ payload }: PayloadReplacerProps) {
                         node.style.setProperty('color', theme.palette.primary.main, 'important')
                     }
                 }}
-                href={payload.startsWith('https://maskbook.com') ? payload : 'https://maskbook.com'}>
+                href={/^https?:\/\/mask(book\.com|\.io)/.test(payload) ? payload : 'https://mask.io'}>
```

Comment:
I suggest using `startsWith` because it is easier to read and maintain. cc @septs